src/test/helpers/pubsub.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { EventEmitter } from "events";
+import type { Message } from "@google-cloud/pubsub";
+
+export function mockMessage<T extends object>(payload: T) {
```

Comment:
Nit: I don't think this generic does anything, since it's not included in the return type :)